components/tabstrip/tabs-configuration.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Tabs Configuration
+page_title: TabStrip Tabs Configuration
+description: Overview of the TabStrip for Blazor.
+slug: tabstrip-tabs-configuration
+tags: telerik,blazor,tab,strip,tabstrip,overview
+published: True
+position: 7
+---
+
+# Tabs Configuration
+
+You can configure the `TabStripTab` component and its child `TabStrip` instances through the following parameters:
+
+* [Title](#title)
+* [Visible](#visible)
+* [Disabled](#disabled)
+
+
+## Title
+
+The `Title` parameter allows you to define the desired text that will be rendered in the Tab heading. If not set, no text will be rendered in the `TabStrip` heading.
+
+>caption Set the desired title for the tab heading.
+
+````CSHTML
+<TelerikTabStrip >
+    <TabStripTab Title="First">
+        First tab content.
+    </TabStripTab>
+    <TabStripTab Title="Second">
+        Second tab content.
+    </TabStripTab>
+    <TabStripTab Title="Third">
+        Third tab content.
+    </TabStripTab>
+</TelerikTabStrip>
+````
+
+## Visible
+
+You can control the visibility of the tabs through the `Visible` parameter the `TabStripTab` exposes. If you toggle a tab visibility at runtime, its order will be preserved.
```

Comment:
We may want to clarify why we mention the note about tab order. When it is not preserved?

```suggestion
Control tab visibility through the `Visible` parameter of the `TabStripTab`. If you toggle the visibility at runtime, the tab order will be preserved.
```

Reply:
The note is not because the order is not preserved is some cases but because this was highly requested. If it is considered confusing, I can remove it.